venues/ICLR.cc/2019/Conference/python/groups.py

```diff
@@ -128,8 +129,14 @@
 }
 
 def create_and_post(client, paper, template_key):
-    client.post_group(openreview.Group.from_json(
-        openreview.tools.fill_template(group_templates[template_key], paper)))
+    group_to_post = openreview.Group.from_json(
+        openreview.tools.fill_template(
+            group_templates[template_key], paper))
+
+    if template_key == "Paper/Authors":
```

Comment:
I would define a new parameter 'members' that is optional. And you should send the members in the post_submissions_stage.